src/scheduler/server_info.c

```diff
@@ -1899,23 +1899,22 @@ copy_server_arrays(server_info *nsinfo, server_info *osinfo)
 		log_err(errno, __func__, "Error allocating memory");
 		return 0;
 	}
-	qinfo = nsinfo->queues;
-	while (*qinfo != NULL) {
+
+	for (qinfo = nsinfo->queues; *qinfo != NULL; qinfo++) {
```

Comment:
I know this code is preexisting, but if you want to be more consistent with the rest of the scheduler use an index instead of ++ing the array.  It's fine if you want to leave it alone.

Reply:
will do